### ISSUE TITLE
Does pydoc-markdown support Python 3.7 asyncio files? Complains about syntax error

### ISSUE THREAD

**Issue by MrYsLab** [NiklasRosenstein/pydoc-markdown#72]
Comment is below - sorry - I hit enter too quickly.

**Comment by MrYsLab** [comment]
I am trying to generate documentation for a [Python 3.7 compatible asyncio file](https://github.com/MrYsLab/pymata-express/blob/master/pymata_express/pymata_express.py) that does not contain syntax errors and runs without issue, but I am getting a "syntax" error from pydoc-markdown. Is the tool compatible with Python 3.7 and asyncio?

Here is my console output:

```
pydocmd simple pymata_express
Building index...
Traceback (most recent call last):
  File "/usr/local/bin/pydocmd", line 11, in <module>
    load_entry_point('pydoc-markdown==2.0.5', 'console_scripts', 'pydocmd')()
  File "/usr/local/lib/python3.7/site-packages/pydocmd/__main__.py", line 229, in main
    add_sections(doc, args.subargs)
  File "/usr/local/lib/python3.7/site-packages/pydocmd/__main__.py", line 194, in add_sections
    [add_sections(doc, x, depth) for x in object_names]
  File "/usr/local/lib/python3.7/site-packages/pydocmd/__main__.py", line 194, in <listcomp>
    [add_sections(doc, x, depth) for x in object_names]
  File "/usr/local/lib/python3.7/site-packages/pydocmd/__main__.py", line 217, in add_sections
    create_sections(object_names, 0)
  File "/usr/local/lib/python3.7/site-packages/pydocmd/__main__.py", line 213, in create_sections
    for sub in dir_object(name, sort_order, need_docstrings):
  File "/usr/local/lib/python3.7/site-packages/pydocmd/imp.py", line 95, in dir_object
    obj = import_object(name)
  File "/usr/local/lib/python3.7/site-packages/pydocmd/imp.py", line 44, in import_object
    return import_object_with_scope(name)[0]
  File "/usr/local/lib/python3.7/site-packages/pydocmd/imp.py", line 64, in import_object_with_scope
    obj = import_module(current_name)
  File "/usr/local/lib/python3.7/site-packages/pydocmd/imp.py", line 36, in import_module
    return __import__(name, fromlist=[''])
  File "./pymata_express.py", line 261
    firmware_version = await self.get_firmware_version()
                      ^
SyntaxError: 'await' outside async function
```

**Comment by NiklasRosenstein** [comment]
Hi @MrYsLab,

well what is the Python version that you installed Pydoc-Markdown into? It imports the Python module to read the docstrings, and it looks like the Python version that you installed Pydoc-Markdown into can not load this module.

On a side note, v3 (in development) won't have that limitation as it parses the source code rather than importing it.

**Comment by MrYsLab** [comment]
Hi @NiklasRosenstein Thanks for the quick reply. I installed using pip3 on Ubuntu linux 18.10 with the following command:
```
sudo pip3 install pydoc-markdown

```
If instead I install using:
```
sudo pip3 install git+https://github.com/NiklasRosenstein/pydoc-markdown.git
```

I get different output:

```
pydocmd simple pymata_express
Traceback (most recent call last):
  File "/usr/local/bin/pydocmd", line 11, in <module>
    load_entry_point('pydoc-markdown==2.0.5', 'console_scripts', 'pydocmd')()
  File "/usr/local/lib/python3.7/site-packages/pydocmd/__main__.py", line 187, in main
    loader = import_object(config['loader'])(config)
  File "/usr/local/lib/python3.7/site-packages/pydocmd/imp.py", line 44, in import_object
    return import_object_with_scope(name)[0]
  File "/usr/local/lib/python3.7/site-packages/pydocmd/imp.py", line 69, in import_object_with_scope
    if '__dict__' in obj:
TypeError: argument of type 'module' is not iterable
```
I am running python 3.7.2

**Comment by NiklasRosenstein** [comment]
Hey @MrYsLab sorry for the late reply. I believe the last error you linked is something that was fixed recently in #75.

**Comment by MrYsLab** [comment]
Yes, it is fixed - thank you.
